Proy/game.py

```diff
@@ -1,81 +1,118 @@
+""""
++Encontre una extension para el estilo PEP8 en Visual Studio Code llamada autopep8 y tambien black formatter.
+-No encontre un atajo ni extension para python para las importaciones
++Mejoré la funcion winner usando set() y evitando mucha repeticion de (== symbol)
++Use try except para que no se rompa el programa cuando no se ingresen numeros en la funcion get_valid_input(game), ademas use .replace(",", " ").split() para que acepte datos separados como: 
+1,1
+1 ,1
+1 , 1
+1, 1
+1 1
+1     1
+*Estoy aprendiendo como funciona pytest y todavia no se como simular una entrada de usuario, logre hacer el test solo a 3 funciones
++averigue que 'if __name__ == "__main__":' sirve para vevitar que partes del codigo se ejecuten cuando se lo importa como modulo ya que cuando se ejecuta un script en python se asigna un valor a la variable __name__ 
+ySi el script se ejecuta directamente (como el programa principal), el valor de __name__ se establece en "__main__".
+Si el script se importa como un módulo, el valor de __name__ se establece al nombre del módulo.
++Para evitar muchas identaciones y muchas lineas en la funcion start() extrai codigo en dos funciones más
+"""
+
+
 import random
-import os, time
+import os
 
 
 def draw(game):
```

Comment:
There is sort of a name-clash, as "draw" can also mean "indeciso/empate". Maybe we should go with ´show´instead